Makefile

```diff
@@ -55,9 +55,11 @@ nostril: $(HEADERS) $(OBJS) $(ARS)
 	$(CC) $(CFLAGS) $(OBJS) $(ARS) -o $@
 
 install: all
-	install -Dm644 doc/nostril.1 $(PREFIX)/share/man/man1/nostril.1
-	install -Dm755 nostril $(PREFIX)/bin/nostril
-	install -Dm755 nostril-query $(PREFIX)/bin/nostril-query
+	mkdir -p $(PREFIX)/share/man/man1
```

Comment:
isnt install supposed to do this?

Reply:
thanks for the review:
I will place it in a separate commit for testing with and without.